src/jit/lsraarmarch.cpp

```diff
@@ -711,6 +711,12 @@ void Lowering::TreeNodeInfoInitPutArgStk(GenTreePutArgStk* argNode, fgArgTabEntr
             MakeSrcContained(argNode, putArgChild);
         }
     }
+    else if (putArgChild->TypeGet() == TYP_DOUBLE) {
```

Comment:
Yes, right. I will add next patch for adding `_TARGET_ARM_` tag and changing logic.